apps/video/view.html

```diff
@@ -27,26 +29,31 @@
             <h1 id="video-title"></h1>
             <button id="save" data-l10n-id="save" hidden></button>
           </gaia-header>
-
-          <footer id="videoBar">
-            <div id="timeSlider">
-              <span id="elapsed-text"></span>
-              <div id="slider-wrapper">
-                <div id="elapsedTime" class="progress"></div>
-                <div id="bufferedTime" class="progress"></div>
-                <div id="timeBackground" class="progress"></div>
-                <button id="playHead"></button>
+          <div id="hide-controls" role="button"
+               data-l10n-id="hide-controls-button"></div>
```

Comment:
Nit: to be consistent, the ending 'div' tag should be on a separate line, lined up with the beginning 'div' tag.